src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/HtmlHelper.cs

```diff
@@ -45,7 +45,8 @@ public class HtmlHelper : IHtmlHelper, IViewContextAware
             IModelMetadataProvider metadataProvider,
             IViewBufferScope bufferScope,
             HtmlEncoder htmlEncoder,
-            UrlEncoder urlEncoder)
+            UrlEncoder urlEncoder,
+            ExpressionTextCache expressionTextCache)
```

Comment:
No reason for `HtmlHelper` to have every constructor parameter that `HtmlHelper<T>` has. Leave this out because the class doesn't need or want the `ExpressionTextCache`.